flathub.json

```diff
@@ -1,4 +1,4 @@
 {
     "only-arches": ["x86_64"],
-    "automerge-flathubbot-prs": true
+    "automerge-flathubbot-prs": false
```

Comment:
Is this because of https://docs.flathub.org/docs/for-app-authors/linter/#flathub-json-automerge-enabled? What is meant by "This is no longer allowed by default."? How can I activate this again? This is a handy feature to auto-merge minor dependency updates.

Reply:
sorry but flathub linter refuse this now. On my side I think automerge is a bad idea

Reply:
I found this blog post https://docs.flathub.org/blog/linter-restricting-automatic-merge/ with the motivation behind this change.